src/batch-submitter/tx-batch-submitter.ts

```diff
@@ -310,6 +349,12 @@ export class TransactionBatchSubmitter extends BatchSubmitter {
       //  In this case, we want to submit regardless of the batch's size.
       wasBatchTruncated = true
     }
+
+    this.log.info('Generated sequencer batch params', {
+      contexts: sequencerBatchParams.contexts,
```

Comment:
How large is this?

Reply:
the contexts is quite small (in the test with 6 transactions), the transactions are actually the longer parts of this log here. i'll keep it and de-bloat if this ends up being too large?